### ISSUE TITLE
Setup script can't install pip packages to the user's environment on Arch Linux

### ISSUE THREAD

**Issue by JacoMalan1** [google/OpenSK#647]
## Expected Behavior
Setup script install pip3 packages into the user's home directory without needing elevated permissions.

## Actual Behavior
The setup script fails with this Output:

```
[-] Copying additional boards to Tock... DONE.
[-] Applying patch "01-persistent-storage.patch"... DONE.
[-] Applying patch "02-usb.patch"... DONE.
[-] Applying patch "03-usb-debugging.patch"... DONE.
[-] Applying patch "04-additional-boards.patch"... DONE.
[-] Applying patch "05-mpu-fix.patch"... DONE.
[-] Applying patch "06-update-uicr.patch"... DONE.
[-] Applying patch "07-firmware-protect.patch"... DONE.
[-] Applying patch "01-remove-test-runner.patch"... DONE.
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try 'pacman -S
    python-xyz', where xyz is the package you are trying to
    install.

    If you wish to install a non-Arch-packaged Python package,
    create a virtual environment using 'python -m venv path/to/venv'.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip.

    If you wish to install a non-Arch packaged Python application,
    it may be easiest to use 'pipx install xyz', which will manage a
    virtual environment for you. Make sure you have python-pipx
    installed via pacman.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

## Steps to Reproduce the Problem

1. Run `./setup.sh`.

## Specifications

- Version: commit 5b67f3f9a8920ed1d503a6772a406fa04727d4b7
- Platform: Arch Linux
```
$ uname -a
Linux REDACTED 6.4.12-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 24 Aug 2023 00:38:14 +0000 x86_64 GNU/Linux
```

**Comment by kaczmarczyck** [comment]
Should be fixed by #653 .
Feel free to test and comment on the PR!

**Comment by kaczmarczyck** [comment]
Please reopen if the issue persists for you.
